### PR TITLE
More RuntimeService tests

### PR DESCRIPTION
**Pull request by vinjana**
+ plus toFile throws ConfigurationError.

### CODE REVIEW

**Comment by vinjana** [comment]
The throw may result in API break! Not sure whether to keep it.

File: RoddyCore/test/src/de/dkfz/roddy/core/RuntimeServiceTest.groovy [hunk 1]
```diff
@@ -6,118 +6,120 @@
...
-    @BeforeClass
-    public static void initialize() {
-        ExecutionService.initializeService(LocalExecutionService, RunMode.CLI)
-        FileSystemAccessProvider.initializeProvider(true)
-        folderOfPids = new File("./test/resources/testpids")
+    @Test
+    void testGetTestAnalysis() {
+        def a = mockedContext.analysis
+        assert a.getInputBaseDirectory().toString() ==
+                inputBaseDirectory.toString().replace('${projectName}', mockedContext.project.name)
```
Comment (dankwart-de) [inline RoddyCore/test/src/de/dkfz/roddy/core/RuntimeServiceTest.groovy]: code repetition!

File: RoddyCore/test/src/de/dkfz/roddy/core/RuntimeServiceTest.groovy [hunk 2]
```diff
@@ -6,118 +6,120 @@
...
+                inputBaseDirectory.toString().replace('${projectName}', mockedContext.project.name) +
+                FileSystemAccessProvider.instance.pathSeparator + mockedContext.dataSet.id
     }
 
     @Test
-    public void testGetTestAnalysis() {
-        def a = getTestAnalysis()
-        assert a.getInputBaseDirectory() == folderOfPids
-        assert a.getOutputBaseDirectory() == folderOfPids
+    void testGetOutputFolderForDataSetAndAnalysis() {
```
Comment (dankwart-de) [inline RoddyCore/test/src/de/dkfz/roddy/core/RuntimeServiceTest.groovy]: Didn't you rename this in the other PR? #210 ?

File: build.gradle [hunk 3]
```diff
@@ -38,7 +38,7 @@ group = "com.github.eilslabs"
```
Comment (dankwart-de) [inline build.gradle]: Can't comment line 31 directly. Please get rid of eilslabs related entries if possible.

File: RoddyCore/test/src/de/dkfz/roddy/core/RuntimeServiceTest.groovy (hunk 1 above)
Comment (vinjana) [reply RoddyCore/test/src/de/dkfz/roddy/core/RuntimeServiceTest.groovy]: Won't fix this. It's just two occurrences. We discussed it.

File: build.gradle (hunk 3 above)
Comment (vinjana) [reply build.gradle]: fixed

Comment (vinjana) [reply build.gradle]: There are lots of copyright notices etc. containing eilslabs. We can change these in one go later.

**Comment by vinjana** [comment]
Code is now included in the refactorings branch. I close this PR.

**Pull request closed without merging**